server/controllers/requestController.js

```diff
@@ -0,0 +1,44 @@
+import RequestHelper from '../helpers/requestHelper';
+import users from '../helpers/authHelpers';
+
+/**
+ * This class contains all methods
+ * required to approve/reject
+ *  atrip requests.
+ */
+class RequestController {
+  /**
+   * This method approve the trip request.
+   * @param {object} req The user's trip request.
+   * @param {object} res The response.
+   * @returns {object} The status and some data of the trip.
+   */
+  static async approveRequest(req, res) {
+    const reqq = await RequestHelper.findRequest('id', parseInt(req.params.id, 10));
+
+    if (!reqq) {
+      res.status(400).json({
+        status: 400,
+        error: 'Please provide a authentic request'
+      });
+    }
+
+    const manager = await users.userExists('id', parseInt(reqq.userId, 10));
```

Comment:
1. what is the difference between manager and properManager line 26,27
2. what shows in the code that we are approving. I only see we are updating the status, maybe we are rejecting